developer/openshift/gitops/argocd/pipeline-service-storage/postgres.yaml

```diff
@@ -14,12 +14,12 @@ spec:
   source:
     chart: postgresql
     repoURL: https://charts.bitnami.com/bitnami
-    targetRevision: 12.2.7
+    targetRevision: 14.0.5
     helm:
       releaseName: postgres
       parameters:
         - name: image.tag
-          value: 13.10.0
+          value: 13.14.0
```

Comment:
where is that value coming from?

Reply:
I went through the bitnami chart index, this is the only one in postgres 13.x.x that is updated. All others are either deprecated or removed.

For the bitnami chart version, I took the latest release since 13.14 was released in the charts in Feb.

Is there a procedure we should follow??

Reply:
I have no expirience with helm, but I did `helm show chart oci://registry-1.docker.io/bitnamicharts/postgresql` and got ```Pulled: registry-1.docker.io/bitnamicharts/postgresql:14.0.5
Digest: sha256:130cf4bff96f75f0cfaa4e7c2fb24d63c5e1f61735fcce10fb230788039109fc
annotations:
  category: Database
  images: |
    - name: os-shell
      image: docker.io/bitnami/os-shell:11-debian-11-r96
    - name: postgres-exporter
      image: docker.io/bitnami/postgres-exporter:0.15.0-debian-11-r9
    - name: postgresql
      image: docker.io/bitnami/postgresql:16.2.0-debian-11-r1
  licenses: Apache-2.0
apiVersion: v2
appVersion: 16.2.0
dependencies:
- name: common
  repository: oci://registry-1.docker.io/bitnamicharts
  tags:
  - bitnami-common
  version: 2.x.x
description: PostgreSQL (Postgres) is an open source object-relational database known
  for reliability and data integrity. ACID-compliant, it supports foreign keys, joins,
  views, triggers and stored procedures.
home: https://bitnami.com
icon: https://bitnami.com/assets/stacks/postgresql/img/postgresql-stack-220x234.png
keywords:
- postgresql
- postgres
- database
- sql
- replication
- cluster
maintainers:
- name: VMware, Inc.
  url: https://github.com/bitnami/charts
name: postgresql
sources:
- https://github.com/bitnami/charts/tree/main/bitnami/postgresql
version: 14.0.5``` and I was wondering shouldn't that tag be `14.0.5`?

Reply:
| I went through the bitnami chart index, this is the only one in postgres 13.x.x that is updated
Can you share the steps to replicate that? That was my initial question.

Reply:
14.0.5 is the chart version, and it is not the same as app version. I too got confused because it was my first time working with helm charts. But after going through many blogs and docs I got that these versions are independent and you can have multiple app versions in a single chart version.

Reply:
I understand there is a difference between chart version and app version and the chart version of `14.0.5` is what we agree on. I'm still trying to understand where did you get the app version from? From my query above, shouldn't that be `16.2.0`. Based on https://www.postgresql.org/docs/release/, the newest version of Postgres indeed is 16.2.

Reply:
Sure I will explain what I did. Unfortunately, there is no easy way to get a chart version for a given app version.
1. first , when using helm cli you can see what all chart versions are available, but I couldn't find a 13.x version

<details>
<summary>bitnami/postgresql helm</summary>

```
$ helm search repo bitnami/postgresql -l                 
NAME                 	CHART VERSION	APP VERSION	DESCRIPTION                                       
bitnami/postgresql   	14.0.5       	16.2.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	14.0.4       	16.2.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	14.0.3       	16.2.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	14.0.2       	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	14.0.1       	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	14.0.0       	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.4.4       	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.4.3       	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.4.2       	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.4.1       	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.4.0       	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.3.1       	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.3.0       	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.30      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.29      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.28      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.27      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.26      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.25      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.24      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.23      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.22      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.21      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.20      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.19      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.18      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.17      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.16      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.15      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.14      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.13      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.12      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.11      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.10      	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.9       	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.8       	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.7       	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.6       	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.5       	16.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.4       	16.0.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.3       	16.0.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.2       	16.0.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.1       	16.0.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.2.0       	16.0.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.1.5       	16.0.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.1.4       	16.0.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.1.2       	16.0.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.1.1       	16.0.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.1.0       	16.0.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	13.0.0       	16.0.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.12.10     	15.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.12.9      	15.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.12.7      	15.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.12.5      	15.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.12.4      	15.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.11.2      	15.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.11.1      	15.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.11.0      	15.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.10.2      	15.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.10.1      	15.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.10.0      	15.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.9.0       	15.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.8.5       	15.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.8.4       	15.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.8.3       	15.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.8.2       	15.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.8.1       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.8.0       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.7.3       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.7.2       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.7.1       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.7.0       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.6.9       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.6.8       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.6.7       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.6.6       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.6.5       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.6.4       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.6.3       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.6.2       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.6.1       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.6.0       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.5.9       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.5.8       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.5.7       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.5.6       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.5.5       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.5.4       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.5.3       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.5.2       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.5.1       	15.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.4.3       	15.2.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.4.2       	15.2.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.4.1       	15.2.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.3.2       	15.2.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.3.1       	15.2.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.2.8       	15.2.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.2.7       	15.2.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.2.6       	15.2.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.2.5       	15.2.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.2.4       	15.2.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.2.3       	15.2.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.2.2       	15.2.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.2.1       	15.2.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.2.0       	15.2.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.1.15      	15.2.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.1.14      	15.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.1.13      	15.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.1.12      	15.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.1.11      	15.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.1.10      	15.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.1.9       	15.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.1.8       	15.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.1.7       	15.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.1.6       	15.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.1.5       	15.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.1.4       	15.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.1.3       	15.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.1.2       	15.1.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.1.1       	15.0.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.1.0       	15.0.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.0.1       	15.0.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	12.0.0       	15.0.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.9.13      	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.9.12      	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.9.11      	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.9.10      	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.9.9       	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.9.8       	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.9.7       	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.9.6       	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.9.5       	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.9.2       	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.9.1       	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.9.0       	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.8.2       	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.8.1       	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.8.0       	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.7.6       	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.7.5       	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.7.4       	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.7.3       	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.7.2       	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.7.1       	14.5.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.7.0       	14.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.26      	14.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.25      	14.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.24      	14.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.23      	14.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.22      	14.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.21      	14.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.20      	14.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.19      	14.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.18      	14.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.17      	14.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.16      	14.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.15      	1.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.14      	14.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.13      	14.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.12      	14.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.11      	14.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.10      	14.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.9       	14.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.8       	14.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.7       	14.4.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.6       	14.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.5       	14.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.4       	14.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.3       	14.3.0     	PostgreSQL (Postgres) is an open source object-...
bitnami/postgresql   	11.6.2       	14.3.0     	PostgreSQL (Postgres) is an open source object-...
```
</details>

2. After that I checked the chart here https://artifacthub.io/packages/helm/bitnami/postgresql, so 14.0.5 chart was released yesterday and that is when all the bitnami/postgresql images were updated as well, the chart doesn't show 13.x thought, it only shows the latest version of postgresql it is released with.
3. Then I went to https://hub.docker.com/r/bitnami/postgresql/tags?page=1&name=13 and searched for 13.x, so we were already using 13.10, but it was not updated, so I chose the next version which was updated, that is 13.14.0
4. Finally I used 14.0.5 as chart version and 13.14.0 as postgresql

There is a little bit of guess work thought

Reply:
yes 16.2 is the latest it shows, we cannot use 16.x though